docs/source/basic_usage.rst

```diff
@@ -85,213 +74,142 @@ File: model.py ::
 
-    #Core Django imports
     from django.test import TestCase
 
-    #Third-party app imports
-    from model_mommy import mommy
-    from model_mommy.recipe import Recipe, foreign_key
+    from model_bakery import baker
 
-    # Relative imports of the 'app-name' package
-
-    class DogTestModel(TestCase):
-        """
-        Class to test the model
-        Dog
-        """
+    class PurchaseHistoryTestModel(TestCase):
 
         def setUp(self):
-            """
-            Set up all the tests
-            """
-            self.rex = mommy.make('family.Dog', make_m2m=True)
+            self.history = baker.make('shop.PurchaseHistory', make_m2m=True)
+            print(self.history.products.count())
+
 
 Explicit M2M Relationships
 --------------------------
-If you want to, you can prepare your own set of related object and pass it to model_mommy. Here's an example:
-
-File: test_models.py ::
-    dogs_set = mommy.prepare(models.Dog, _quantity=2)
-    home = mommy.make(models.Home, owner=owner, dogs=dogs_set)
+If you want to, you can prepare your own set of related object and pass it to Model Bakery. Here's an example: ::
 
+    products_set = baker.prepare(Product, _quantity=5)
+    history = baker.make(PurchaseHistory, products=products_set)
 
-Defining some attributes
-------------------------
 
-Of course it's possible to explicitly set values for attributes.
-
-File: test_model.py ::
+Explicit values for fields
+--------------------------
 
-    # -*- coding:utf-8 -*-
+By default, Model Bakery uses random values to populate the model's fields. But it's possible to explicitly set values for them as well. ::
 
-    #Core Django imports
     from django.test import TestCase
 
-    #Third-party app imports
-    from model_mommy import mommy
-    from model_mommy.recipe import Recipe, foreign_key
-
-    # Relative imports of the 'app-name' package
-    from .models import Kid
+    from model_bakery import baker
 
-    class KidTestModel(TestCase):
-        """
-        Class to test the model
-        Kid
-        """
+    class CustomerTestModel(TestCase):
 
         def setUp(self):
-            """
-            Set up all the tests
-            """
-            self.kid = mommy.make(
-                Kid,
-                age=3
+            self.customer = baker.make(
+                'shop.Customer',
+                age=21
             )
 
-            self.another_kid = mommy.make(
-                'family.Kid',
-                age=6
+            self.older_customer = baker.make(
+                'shop.Customer',
+                age=42
             )
 
-Related objects attributes are also reachable by their name or related names:
-
-File: test_model.py ::
-
-    # -*- coding:utf-8 -*-
+Related objects fields are also reachable by their name or related names in a very similar way as Django does with `field lookups <https://docs.djangoproject.com/en/2.2/ref/models/querysets/#id4>`_: ::
```

Comment:
```suggestion
Related objects fields are also reachable by their name or related names in a very similar way as Django does with `field lookups <https://docs.djangoproject.com/en/dev/ref/models/querysets/#field-lookups>`_: ::
```

What about putting `dev` version links, so it would not get outdated? In case, we will have some mismatch with possible new functionality, we better add new Django version support.
Also, I changed anchor to a verbose one.